idunn/blocks/services_and_information.py

```diff
@@ -23,9 +22,14 @@ def from_es(cls, es_poi, lang):
         properties = es_poi.get("properties", {})
 
         raw_wheelchair = properties.get("wheelchair")
-        raw_tactile_paving = properties.get("tactile_paving")
         raw_toilets_wheelchair = properties.get("toilets:wheelchair")
 
+        if all(
+            s is None
+            for s in (raw_wheelchair, raw_toilets_wheelchair)
+        ):
+            return None
```

Comment:
The check is redundant and can be removed.

Reply:
your are right: removed